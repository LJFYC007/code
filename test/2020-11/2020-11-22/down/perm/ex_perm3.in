2
19
18 8 12 3 1 2 5 11 17 7 16 9 13 14 19 4 6 15 10
6 19 17 10 8 9 7 4 14 18 13 1 12 3 15 11 16 2 5
19
13 18 7 16 12 19 9 2 4 11 10 14 3 6 1 8 5 17 15
8 4 13 5 3 2 12 7 10 15 18 16 11 19 6 9 17 14 1
