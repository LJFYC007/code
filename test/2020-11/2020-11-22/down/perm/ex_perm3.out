608407547
582459399
